app-doc/heirloom-doctools/heirloom-doctools-160308.ebuild

```diff
@@ -0,0 +1,87 @@
+EAPI=6
+
+inherit flag-o-matic toolchain-funcs multilib readme-gentoo
+
+DESCRIPTION="Classic Unix documentation tools ported from OpenSolaris"
+HOMEPAGE="https://n-t-roff.github.io/heirloom/doctools.html"
+SRC_URI="https://github.com/n-t-roff/heirloom-doctools/releases/download/${PV}/${P}.tar.bz2"
+
+LICENSE="CDDL"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cxx"
+
+RDEPEND="!sys-apps/groff"
+DEPEND="sys-devel/flex
+	virtual/yacc"
+
+src_prepare() {
+	# mpm uses C++, we'll build it explicitly if we really want to
+	sed -i -e 's:mpm:$(MPM):' makefile || die
```

Comment:
Didn't I suggest using a patch for all of those changes? seds are really hard to maintain since they don't fail if the substitution doesn't match or replaces too much.